code/Data exploration.R

```diff
@@ -282,15 +381,112 @@ MAE_MSE <- data.frame(Error_measure,Values)
 rm(MAE,MSE,Error_measure,Values)
 print(MAE_MSE)
 
-##Create boxplot of total demand by time 
+##Summary of measures from key tables
+summary(data)
+
+###########
+##Boxplot##
+###########
+
+##Create boxplot of total demand by time (year-month)
+ggplot(data) + geom_boxplot(aes(x = YEARMONTH, y = TOTALDEMAND, group = YEARMONTH))
```

Comment:
will need to polish this for the report (include title, axis labels, etc.)

Reply:
Agreed, I'll do the polishing tonight.